llvm/include/llvm/CodeGen/MachineBasicBlock.h

```diff
@@ -846,8 +846,11 @@ class MachineBasicBlock
 
   /// Return the first instruction in MBB after I that is not a PHI, label or
   /// debug.  This is the correct point to insert copies at the beginning of a
-  /// basic block.
-  iterator SkipPHIsLabelsAndDebug(iterator I, bool SkipPseudoOp = true);
+  /// basic block. \p Reg is an optional argument passed during register
+  /// allocator to have additional target specific checks for its spill/copy
```

Comment:
allocator -> allocation. I also believe it shall be spill/restore/copy.

Reply:
This is still a roundabout way of saying what the register value is. Say something that expresses this is the register being defined for a spill/split

Reply:
I'm thinking this should be a differently named function, specifically for the spill insert point